andela/ah-frontend-valkyrie
# #162633459 Implement Reset Password Functionality

**What does this PR do?**

Implements reset password functionality.

**Description of Task to be completed?**
- Create components for sending email and reseting password
- Create actions 
- Create reducers
- Test the actions, reducers and components.

**How should this be manually tested?**
- Run `npm install` to install any dependencies
- Run `npm test` to check for testing.
- Run `npm start` to run the application
- On the UI click `sign in` and if you forgot your password click `reset password`

**What are the relevant pivotal tracker stories?**
[#162633459](https://www.pivotaltracker.com/story/show/162633459)
